File: python/agentic_commerce_protocol/models/payment_provider.py
# this file is @generated
import typing as t

from .common import BaseModel
from .payment_method_type import PaymentMethodType
from .payment_provider_type import PaymentProviderType


class PaymentProvider(BaseModel):
    provider: PaymentProviderType
    supported_payment_methods: t.List[PaymentMethodType]

File: kotlin/src/main/kotlin/com/svix/agentic_commerce_protocol/PaymentProvider.kt
// This file is @generated
package com.svix.agentic_commerce_protocol

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class PaymentProvider(
    val provider: PaymentProviderType,
    @SerialName("supported_payment_methods") val supportedPaymentMethods: List<PaymentMethodType>,
)
